### ISSUE TITLE
Should “es6”` "es2016"?

### ISSUE THREAD

**Issue by steshaw** [pulumi/docs#3878]
File: [docs/intro/languages/javascript.md](https://www.pulumi.com/docs/intro/languages/javascript/)

Wondering if `target: “es6”` be `target: "es2016"`?

I have `target: "es2016"` in my `tsconfig.json` file and Pulumi seems to work just fine. Should I switch to "es6" or do the docs need an update?

**Comment by lukehoban** [comment]
Yep. The templates have been updated to use `es2016` so the docs should be updated to match as well. PRs welcome!

**Comment by steshaw** [comment]
Turns out that someone has raised a PR for "es6" -> "es2016"

https://github.com/pulumi/docs/pull/3024

It's from April and looks like it needs a little shepherding.
